streaming/__init__.py

```diff
@@ -1,10 +1,15 @@
 # Copyright 2022 MosaicML Streaming authors
 # SPDX-License-Identifier: Apache-2.0
 
+"""MosaicML Streaming Datasets for cloud-native model training."""
+
+import streaming.text as text
+import streaming.vision as vision
 from streaming._version import __version__
 from streaming.base import (CSVWriter, Dataset, JSONWriter, LocalDataset, MDSWriter, TSVWriter,
                             XSVWriter)
 
 __all__ = [
-    'Dataset', 'CSVWriter', 'JSONWriter', 'MDSWriter', 'TSVWriter', 'XSVWriter', 'LocalDataset'
+    'Dataset', 'CSVWriter', 'JSONWriter', 'MDSWriter', 'TSVWriter', 'XSVWriter', 'LocalDataset',
+    'vision', 'text'
```

Comment:
i assume we want `base` too here

Reply:
I tried that too but the docs is not generating for sub-directory. I will take a look in a separate PR to make it to work.